### PR TITLE
Expose wallet rebalance config

### PR DESCRIPTION
**Pull request by iturricf**
Exposes the wallet rebalancing feature from the wallet-monitor.

### CODE REVIEW

**Review by solanoepalacio — CHANGES_REQUESTED** [review]
It's really important that we support different rebalance configurations for different chains. We might want to configure them differently depending on how their wallets work (for example SUI doesn't need rebalancing if I understand correctly) and depending the cost associated to rebalancing (we will probably start with no rebalancing enabled for Ethereum). Also, the thresholds will very likely be different for the different chains

File: relayer/middleware/wallet/wallet-management.ts
Comment (solanoepalacio) [inline relayer/middleware/wallet/wallet-management.ts:1]: I think we should probably import WalletRebalanceConfig from wallet-monitor

Comment (iturricf) [reply relayer/middleware/wallet/wallet-management.ts:1]: yes, that's a good idea, I'll complete that once this is published: https://github.com/XLabs/wallet-monitor/pull/77

**Comment by iturricf** [comment]
> It's really important that we support different rebalance configurations for different chains. We might want to configure them differently depending on how their wallets work (for example SUI doesn't need rebalancing if I understand correctly) and depending the cost associated to rebalancing (we will probably start with no rebalancing enabled for Ethereum). Also, the thresholds will very likely be different for the different chains

Yes, that totally makes sense, I've just pushed some changes to do it that way

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -39,4 +39,5 @@ jobs:
         with:
           dry: True
           prettier_options: --write .
+          prettier_version: 2.8.4
```
Comment (solanoepalacio) [inline .github/workflows/test.yaml:42]: did this solve the issue you were having on friday?

File: relayer/middleware/wallet/wallet-management.ts
Comment (solanoepalacio) [reply relayer/middleware/wallet/wallet-management.ts:1]: done! I just published it

File: .github/workflows/test.yaml (hunk 1 above)
Comment (iturricf) [reply .github/workflows/test.yaml]: yes, this solved it. I just added a comment to it

File: relayer/middleware/wallet/wallet-management.ts
Comment (iturricf) [reply relayer/middleware/wallet/wallet-management.ts:1]: thanks. Now I'm using the types from `wallet-monitor`

**Comment by gabzim** [comment]
Should this be closed?

**Pull request closed without merging**